rollup.config.js

```diff
@@ -34,6 +34,15 @@ const commonPlugins = [
   commonjs(),
 ];
 
+const replaceGlobalPlugins = [
```

Comment:
I think rollup has a config option for this: https://rollupjs.org/guide/en/#outputglobals